src/metabase/api/activity.clj

```diff
@@ -112,11 +114,33 @@
 (def ^:private views-limit 8)
 (def ^:private card-runs-limit 8)
 
-(api/defendpoint GET "/recent_views"
+(api/defendpoint ^:deprecated GET "/recent_views"
   "Get a list of 100 models (cards, models, tables, dashboards, and collections) that the current user has been viewing most
   recently. Return a maximum of 20 model of each, if they've looked at at least 20."
   []
-  {:recent_views (recent-views/get-list *current-user-id*)})
+  (u/snake-keys
+   (recent-views/get-recents *current-user-id* :views)))
+
+(api/defendpoint GET "/recents"
+  "Get a list of recent items the current user has been viewing most recently.
+  Allows for filtering by context, either view or selection"
+  [:as {{:keys [context]} :params}]
+  {context [:enum :views :selections :all]}
+  (u/snake-keys
+   (recent-views/get-recents *current-user-id* context)))
+
+(api/defendpoint POST "/recents"
+  "Adds a model to the list of recently viewed or selected items."
+  [:as {{:keys [model model_id context]} :body}]
+  {model (into [:enum] recent-views/models-of-interest)
+   model_id ms/PositiveInt
+   context [:enum :selection]}
```

Comment:
Should this also include `:views` or is the docstring wrong?

Reply:
Docstring was wrong